### DISCUSSION TITLE
Can anyone please tell me step by step process to start this assignment

### DISCUSSION THREAD

**Discussion by vijaypotnuru** [open-sauced-craftwork/browser-extension#18] [Q&A] [answered]
Can anyone please tell me step by step process to start this assignment

**Comment by bdougie** [comment, marked as answer]
@vijaypotnuru there is a link in the [README](https://github.com/open-sauced-craftwork/browser-extension#task-instructions) with task instructions. Can you ask a more clarifying question?

**Reply by vijaypotnuru** [reply]
Should I create new create new project file or clone this main branch file

**Reply by bdougie** [reply]
Clone the repo created for you when you initialized the assignment in Classroom. Run the commands in the [CRXJS](https://crxjs.dev/vite-plugin/getting-started/vanilla-js/create-project) to get the initial extension running. 

At that point you should have enough add the button and check the api for existing users.

**Reply by vijaypotnuru** [reply]
ok thank you sir🙂

**Comment by Sapnajha19** [comment]
I don't understand how to start. Am I supposed to clone the browser extension repo first? [this](https://github.com/open-sauced-craftwork/open_sauced_craftwork_browser_extension-Sapnajha19) or do I have to start by running the commands in the [CRXJS](https://crxjs.dev/vite-plugin/getting-started/vanilla-js/create-project). Can you please explain step by step? And where is the classroom where we have to initialize the assignment?
@bdougie

**Reply by bdougie** [reply]
Classroom - https://classroom.github.com/a/RQ9101eG
Run the CRJXS commands in the repo that is created when accept the assignment.

**Reply by harshal015** [reply]
hey @bdougie I have added some more code and new features to my submission and re-requested review to my pull request which comes with added features.. can you please take a look at it? you can find a detailed description on features with the PR as well.. https://github.com/open-sauced-craftwork/open_sauced_craftwork_browser_extension-Harshal2502/pull/2
